federationapi/queue/queue.go

```diff
@@ -247,9 +247,10 @@ func (oqs *OutgoingQueues) SendEvent(
 		return fmt.Errorf("sendevent: oqs.db.StoreJSON: %w", err)
 	}
 
+	destQueues := []*destinationQueue{}
```

Comment:
`destQueues := make([]*destinationQueue, 0, len(destmap))` is a good way to prevent reallocations here.

Reply:
Same in `SendEDU` too.

Reply:
Good catch